pages/linux/alpine.md

```diff
@@ -0,0 +1,16 @@
+# alpine
+
+> A easy-to-use Email and Internet News program.
```

Comment:
Yeah, when i got this to work about a year ago, there where a lot of looking at guides explaining how to setup email in general on Alpine Linux =)
but hey when you are running Linux you're your own tech support.